code/modules/events/prison_break.dm

```diff
@@ -6,7 +6,28 @@
 	var/list/area/prisonAreas = list()
 
 /datum/event/prison_break/can_start()
-	return 25
+	var/foundSomeone = FALSE
+	var/foundBasic = TRUE
+	for(var/area/A in areas)
+		if(istype(A, /area/security/prison) || istype(A, /area/security/brig))
+			prisonAreas += A
+			var/list/areaMobs = mobs_in_area(A)
+			if(areaMobs && areaMobs.len)
+				foundBasic = TRUE
+			for(var/mob/living/carbon/human/H in areaMobs)
+				var/list/access = H.GetAccess()
+				if(!(access_brig in access))
```

Comment:
unless you check if they're alive, you can still trigger greytide by storing a dead body in evidence storage

Reply:
Fixed